lib/docker-image-tree.coffee

```diff
@@ -15,7 +15,7 @@ exports.createTree = createTree = (images) ->
 		parentId = image.ParentId or root
 		parent = tree[parentId] ?= createNode(parentId)
 
-		node.repoTags = saneRepoTags(image.RepoTags)
+		node.repoTags = if image.RepoTags? then saneRepoTags(image.RepoTags) else '<No tag>'
```

Comment:
Setting `repoTags` to a string can do weird things when it's expected to be an array, eg below you do
```coffee
"Removing image: #{image.repoTags[0]}"
```
which would resolve to `Removing Image <`.

I think it would be much better to instead default to the image id, eg
```coffee
 saneRepoTags = (image) ->
	if not image.RepoTags? or '<none>:<none>' in image.RepoTags 
		return [ image.id ]
	else
		return image.RepoTags 

...

node.repoTags = saneRepoTags(image)
```
that way we get the most useful output possible, unless it breaks https://github.com/resin-io/docker-storage-gc/blob/master/lib/lru.coffee#L36-L39 in which case it needs to stay as an empty array and handle using a nicer identifier another way